pkg/skaffold/deploy/util_test.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deploy
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/deploy/kubectl"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+var (
+	deploymentYaml = `apiVersion: extensions/v1beta1
```

Comment:
The actual content of the yamls does not really matter for this test, right? In that case, I would prefer a very simplistic yaml instead of such a full-blown resource. WDYT?

Reply:
yes.  This is also used in helm_test.go and this is the output of `helm get manifest` for the example repo.

Reply:
done! This was more for testing. I made the yaml slim